app/views/ptables/_custom_tabs.html.erb

```diff
@@ -1,11 +1,7 @@
-<div class="tab-pane" id="template_type">
   <%= checkbox_f f, :snippet, :onchange => "snippet_changed(this)", :label=>_('Snippet'), :disabled => @template.locked? %>
```

Comment:
Can you move `Snippet` and `Operating system` family below `Default`? They are critical fields yet they are hidden under `Audit comment` and the upload Template button, which are much less important.